webextensions/api/webNavigation.json

```diff
@@ -654,7 +654,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/webNavigation/onHistoryStateUpdated",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "22"
```

Comment:
✅ https://github.com/chromium/chromium/commit/fb2a5673c45ffbe71263c6c7cb50fdc57b7dc4aa in [22.0.1210.0](https://github.com/chromium/chromium/releases/tag/22.0.1210.0) before 22.0.1229.